packages/@aws-cdk/aws-kinesisfirehose-destinations/test/s3-bucket.test.ts

```diff
@@ -28,6 +29,10 @@ describe('S3 destination', () => {
     expect(stack).toHaveResource('AWS::KinesisFirehose::DeliveryStream', {
       ExtendedS3DestinationConfiguration: {
         BucketARN: stack.resolve(bucket.bucketArn),
+        BufferingHints: {
+          IntervalInSeconds: 300,
+          SizeInMBs: 5,
+        },
```

Comment:
So, right now, we render the `BufferingHints` property always in the template.

Should we instead render it only if at least one of the properties was provided, and leave it out otherwise?

Reply:
Good idea, will leave it out. It is possible that Firehose will change the defaults here I suppose.